packages/account/src/Sections/Security/ConnectedApps/connected-apps-empty.tsx

```diff
@@ -2,31 +2,25 @@ import React from 'react';
 import { Text } from '@deriv/components';
 import { observer, useStore } from '@deriv/stores';
 import { Localize } from '@deriv/translations';
-import { CONNECTED_APPS_INFO_BULLETS } from 'Constants/connected-apps-config';
+import ConnectedAppsInfoBullets from './connected-apps-info-bullets';
 
 const ConnectedAppsEmpty = observer(() => {
     const { ui } = useStore();
     const { is_mobile } = ui;
 
-    const text_size = is_mobile ? 'xxxs' : 'xxs';
+    const title_text_size = is_mobile ? 'xxs' : 'xs';
+    const bullets_text_size = is_mobile ? 'xxxs' : 'xxs';
```

Comment:
Style file can handle this!

Reply:
i have removed prop drilling from the code and passed `text_size` as prop to `Text` within the component.
please let me know if this needs to be changed.